Bottom: 0a2296f2e8de80e0e20e79733d71b3a52065d2bd
Top:    8d61e72c859eba882a3f2d5c5b34ca2f73d66d06
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2013-07-20 21:25:04 -0700

Refresh of ex-afib

---

diff --git a/examples/udaddy.c b/examples/udaddy.c
index e49c93b..031b0b6 100644
--- a/examples/udaddy.c
+++ b/examples/udaddy.c
@@ -429,7 +429,7 @@ static int alloc_nodes(void)
 		if (dst_addr) {
 			ret = rdma_create_id(test.channel,
 					     &test.nodes[i].cma_id,
-					     &test.nodes[i], port_space);
+					     &test.nodes[i], hints.ai_port_space);
 			if (ret)
 				goto err;
 		}
@@ -644,7 +644,7 @@ int main(int argc, char **argv)
 		case 'P':
 			if (!strncasecmp("ib", optarg, 2)) {
 				hints.ai_port_space = RDMA_PS_IB;
-			} else if (!stncasecmp("ipoib", optarg, 5)) {
+			} else if (!strncasecmp("ipoib", optarg, 5)) {
 				hints.ai_port_space = RDMA_PS_IPOIB;
 			} else if (strncasecmp("udp", optarg, 3)) {
 				fprintf(stderr, "Warning: unknown port space format\n");
